rocky/tools/forms/ooi_form.py

```diff
@@ -63,6 +63,8 @@ def generate_form_fields(
                 fields[name] = generate_ip_field(field)
             elif annotation == AnyUrl:
                 fields[name] = generate_url_field(field)
+            elif annotation == Dict or annotation == Dict[str, str] or annotation == List[str]:
```

Comment:
This should be good enough, it would be nicer to use `get_origin` and `get_args`:
```python3
origin = get_origin(annotation)
args = get_args(annotation)
if origin == dict or (origin == list and args == (str,)): ...
```

Reply:
Could also be picked up later on as there are more inconsistencies within this part of the code base

Reply:
Ideally we'd move all of this 'type'-based rendering into a separate method, which also maps to the frontend field-types and their templating/classing based on what manon structures needs to be used to render such a field.

Reply:
Agree, that could open up possibilities like rendering a org tag field where you can choose already existing tags.